hw2/05-form.js

```diff
@@ -1 +1,13 @@
-// Add your code here
+document.addEventListener("DOMContentLoaded", () => {
+    const form = document.getElementById('userForm');
+    const modal_element = document.getElementById('responseModal');
+
+    const handleSubmit = (event) => {
+        //Prevent page reload on submit
+        event.preventDefault();
+
+        console.log('Event hit');
+    }
+
+    form.addEventListener('submit', handleSubmit);
+})
```

Comment:
When running the **WAVE** in browser it is throwing **1 alert**

**1 Alerts**
1 X Skipped heading level
Skipped heading level 1More information